baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferSafeLoggingPreconditions.java

```diff
@@ -57,7 +59,16 @@
                             .withNameMatching(Pattern.compile("checkArgument|checkState|checkNotNull")),
                     MethodMatchers.staticMethod()
                             .onClass("java.util.Objects")
-                            .named("requireNonNull"));
+                            .named("requireNonNull"),
+                    MethodMatchers.staticMethod()
+                            .onClass("org.apache.commons.lang3.Validate")
+                            .withNameMatching(Pattern.compile("isTrue|notNull|validState")));
+
+    private static final Map<String, String> METHOD_TRANSLATIONS = ImmutableMap.of(
+            "requireNonNull", "checkNotNull",
+            "isTrue", "checkArgument",
+            "notNull", "checkNotNull",
+            "validState", "checkState");
```

Comment:
nit: would prefer to add comments with each entry describing which utility class the method comes from